### PR TITLE
fix: fix for limit-orders unsupported chain

### LINKED ISSUE #2477: Limit: The app is crashed when WETH/ETH token is selected, switch network to unsupported
1. Open Limit orders page
2. Connect to a wallet (MM in ethereum)
3. Select WETH-ETH token pair, an amount
4. Change a network in MM to an aunsupported one (I selected Avalanche)

**AR**: the app is crashed

**ER**: the app is not crashed

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes #2477

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]
LGTM!

File: src/cow-react/common/services/getQuoteCurrency/index.ts [hunk 1]
```diff
@@ -24,11 +25,13 @@ const STABLE_COINS: { [key in SupportedChainId]: string[] } = {
...
 export function getQuoteCurrency(
-  chainId: SupportedChainId | undefined,
+  _chainId: SupportedChainId | undefined,
   inputCurrencyAmount: CurrencyAmount<Currency> | null,
   outputCurrencyAmount: CurrencyAmount<Currency> | null
 ): Currency | null {
-  if (!chainId || !inputCurrencyAmount || !outputCurrencyAmount) return null
+  const chainId = supportedChainId(_chainId)
+
+  if (chainId || !inputCurrencyAmount || !outputCurrencyAmount) return null
```
Comment (alfetopito) [inline src/cow-react/common/services/getQuoteCurrency/index.ts:34]: Shouldn't it be this?

```suggestion
  if (!chainId || !inputCurrencyAmount || !outputCurrencyAmount) return null
```

File: src/cow-react/common/services/getQuoteCurrency/index.ts [hunk 2]
```diff
@@ -41,11 +44,13 @@ export function getQuoteCurrency(
...
 export function getQuoteCurrencyByStableCoin(
-  chainId: SupportedChainId | undefined,
+  _chainId: SupportedChainId | undefined,
   inputCurrency: Currency | null,
   outputCurrency: Currency | null
 ): Currency | null {
-  if (!chainId || !inputCurrency || !outputCurrency) return null
+  const chainId = supportedChainId(_chainId)
+
+  if (!chainId || !supportedChainId(chainId) || !inputCurrency || !outputCurrency) return null
```
Comment (alfetopito) [inline src/cow-react/common/services/getQuoteCurrency/index.ts:53]: Do you need to check both? Also, you are using the return of `supportedChainId()` and running through the same fn again

 ```suggestion
  if (!chainId || !inputCurrency || !outputCurrency) return null
```

**Review by alfetopito — COMMENTED** [review]
I'm a little confused with the changes

**Comment by nenadV91** [comment]
@alfetopito All valid points. Not sure how I missed those but its fixed now.

**Review by alfetopito — APPROVED** [review]

**Pull request merged**